src/mobile/src/ui/components/PasswordFields.js

```diff
@@ -112,7 +107,7 @@ class PasswordFields extends Component {
         return (
             <View style={[styles.container]}>
                 <CustomTextInput
-                    label={passwordLabel}
+                    label={passwordLabel || i18next.t('global:password')}
```

Comment:
We should use the `t` from the props instead (also need `const { t } = this.props;` on line 103):
```suggestion
                    label={passwordLabel || t('global:password')}
```